Simplify the XML-JSON conversion BBE (#2965)
ballerina-platform/ballerina-distribution: examples/xml-to-json-conversion/xml_to_json_conversion.md

# XML JSON conversion

> The `xmldata` library provides API to perform conversion from XML to JSO.

Reviewer: ```suggestion
The `xmldata` library provides API to perform conversion from XML to JSON.
```

> The `xmldata` library provides API to perform conversion from XML to JSON.

Reviewer 2: ```suggestion
The `xmldata` library provides an API to perform conversions from XML to JSON.
```

> For more information on the underlying module, see the [xmldata module](https://lib.ballerina.io/ballerina/xmldata/latest/).

Reviewer 2: ```suggestion
For more information on the underlying module, see the [`xmldata` module](https://lib.ballerina.io/ballerina/xmldata/latest/).
```

::: code xml_to_json_conversion.bal :::

> To run this sample use the `bal run` command.

Reviewer 2: ```suggestion
To run this sample, use the `bal run` command.
```
